### PR TITLE
Support static usage class Enforcer, optimize test case

### PR DESCRIPTION
**Pull request by hlhill**

### CODE REVIEW

**Comment by hlhill** [comment]
Support static usage class Enforcer, optimize test case

File: composer.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
     ],
     "require": {
         "php": "^7.2",
-        "ext-swoole": ">=4.5",
-        "casbin/casbin": "^2.2",
+        "ext-swoole": ">=4.4",
+        "casbin/casbin": "~3.0",
         "easyswoole/orm": "^1.4",
-        "easyswoole/easyswoole": "~3.3|~3.4"
+        "easyswoole/easyswoole": "~3.3|~3.4",
```
Comment (Player626) [inline composer.json]: Delete a comma

Comment (gaobinzhan) [reply composer.json]: Compress your submission into a

**Comment by Player626** [comment]
@hlhill Compress your submission into a

File: tests/DatabaseAdapterTest.php [hunk 2]
```diff
@@ -12,6 +12,23 @@
 
 class DatabaseAdapterTest extends TestCase
 {
+
+    public function setUp()
```
Comment (Player626) [inline tests/DatabaseAdapterTest.php]: Increase the return value `void`

**Comment by Player626** [comment]
@techoner review

**Comment by Player626** [comment]
@techoner You see what else you need to pay attention to. I have read it once and can merge it.

**Comment by hsluoyz** [comment]
@techoner

**Comment by leeqvip** [comment]
@hlhill Please modify commit messsge, refer to: https://github.com/semantic-release/semantic-release#commit-message-format

**Comment by hsluoyz** [comment]
@hlhill plz fix conflicts.

File: composer.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
     ],
     "require": {
         "php": "^7.2",
-        "ext-swoole": ">=4.5",
-        "casbin/casbin": "^2.2",
+        "ext-swoole": ">=4.4",
+        "casbin/casbin": "~3.0",
```
Comment (Player626) [inline composer.json:14]: plz fix conflicts.

File: dev.php [hunk 4]
```diff
@@ -34,9 +34,14 @@
         'host'          => '127.0.0.1',
         'port'          => 3306,
         'user'          => 'root',
-        'password'      => '',
-        'database'      => 'easyswoole_permission',
+        'password'      => 'root',
```
Comment (Player626) [inline dev.php:37]: Automated test failed

Comment (Player626) [reply dev.php:37]: Modify

File: phpunit.php [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+defined("MYSQL_HOST") ?: define('MYSQL_HOST', '10.10.28.130');
+defined("MYSQL_PORT") ?: define('MYSQL_PORT', 3306);
+defined("MYSQL_USER") ?: define('MYSQL_USER', 'root');
+defined("MYSQL_PASSWORD") ?: define('MYSQL_PASSWORD', 'root');
+defined("MYSQL_DATABASE") ?: define('MYSQL_DATABASE', 'test');
+defined("MYSQL_TIMEOUT") ?: define('MYSQL_TIMEOUT', 5);
+defined("MYSQL_CHARSET") ?: define('MYSQL_CHARSET', 'utf8mb4');
```
Comment (Player626) [inline phpunit.php]: Modify

**Comment by hsluoyz** [comment]
@hlhill remove merge commits:

image

File: composer.json [hunk 6]
```diff
@@ -10,10 +10,10 @@
     ],
     "require": {
         "php": "^7.2",
-        "ext-swoole": ">=4.5",
-        "casbin/casbin": "~3.1",
+        "ext-swoole": ">=4.4",
+        "casbin/casbin": "~3.0",
         "easyswoole/orm": "^1.4",
-        "easyswoole/easyswoole": "~3.3|~3.4"
+        "easyswoole/easyswoole": "~3.3|~3.4",
```
Comment (Player626) [inline composer.json]: error

**Comment by Player626** [comment]
@hsluoyz @techoner direct squash and merge

**Comment by hsluoyz** [comment]
Done.

**Pull request merged**